examples/BlynkIrRemote/BlynkIrRemote.ino

```diff
@@ -0,0 +1,196 @@
+18:16:07.597 -> uint32_t address = 0x80;
+18:16:07.597 -> uint32_t command = 0x5;
+18:16:07.597 -> uint64_t data = 0x1FEA05F;
+
+//Song Forward
+
+18:17:20.541 -> Protocol  : NEC
+18:17:20.541 -> Code      : 0x1FEE01F (32 Bits)
+18:17:20.575 -> uint16_t rawData[71] = {9068, 4424,  632, 506,  630, 506,  632, 508,  606, 532,  632, 506,  630, 508,  630, 508,  632, 1620,  632, 1620,  632, 1620,  604, 1646,  606, 1646,  630, 1622,  604, 1646,  632, 1620,  606, 534,  630, 1622,  604, 1646,  630, 1622,  604, 534,  630, 508,  604, 534,  606, 534,  630, 508,  630, 508,  606, 534,  606, 532,  630, 1622,  604, 1646,  632, 1620,  604, 1648,  604, 1646,  604, 39040,  9040, 2216,  604};  // NEC 1FEE01F
+18:17:20.610 -> uint32_t address = 0x80;
+18:17:20.610 -> uint32_t command = 0x7;
+18:17:20.610 -> uint64_t data = 0x1FEE01F;
+
+ */
+
+// check complete video tutorial here for program explanation https://www.youtube.com/watch?v=LqmkDKu54XY&t=17s
+
+/* Comment this out to disable prints and save space */
+#define BLYNK_PRINT Serial
+
+#if defined(ESP8266)
+#include <ESP8266WiFi.h>
+#include <BlynkSimpleEsp8266.h>
+#else
+#include <WiFi.h>
+#endif  // ESP8266
+#if defined(ESP32)
+#include <BlynkSimpleEsp32.h>
+#endif  // ESP32
+
+// IR library
+#include <IRremoteESP8266.h>
+#include <IRsend.h>
+
+const uint16_t kIrLed = 4;  // ESP8266 GPIO pin to use. Recommended: 4 (D2).
+IRsend irsend(kIrLed);  // Set the GPIO to be used to sending the message.
+
+// You should get Auth Token in the Blynk App.
+// Go to the Project Settings (nut icon).
+char auth[] = "YourAuthToken";
+
+// Your WiFi credentials.
+// Set password to "" for open networks.
+char ssid[] = "YourNetworkName";
+char pass[] = "YourPassword";
+
+  BLYNK_WRITE(V51) {  // Power button
+    if (param.asInt() == 0) {
+      // Serial.println("NEC");
+      irsend.sendNEC(0x1FE50AF);
+    }
+  }
+
+  BLYNK_WRITE(V52) {  // Mute button
+    if (param.asInt() == 0) {
+      // Serial.println("NEC");
+      irsend.sendNEC(0x1FE30CF);
+    }
+  }
+
+  BLYNK_WRITE(V53) {  // Song Forward
+    if (param.asInt() == 0) {
+      // Serial.println("NEC");
+      irsend.sendNEC(0x1FEE01F);
+    }
+  }
+
+  BLYNK_WRITE(V54) {  // Song Backward
+    if (param.asInt() == 0) {
+      // Serial.println("NEC");
+      irsend.sendNEC(0x1FEA05F);
+      delay(10);  // double tap back button to back one song
+      irsend.sendNEC(0x1FEA05F);
+    }
+  }
+
+  BLYNK_WRITE(V55) {  // Volume --
+    if (param.asInt() == 0) {
+      // Serial.println("NEC");
+      irsend.sendNEC(0x1FEC03F);
+    }
+  }
+
+  BLYNK_WRITE(V56) {  // Volume ++
+    if (param.asInt() == 0) {
+      // Serial.println("NEC");
+      irsend.sendNEC(0x1FE40BF);
+    }
+  }
+
+  BLYNK_WRITE(V57) {  // Play/Pause
+    if (param.asInt() == 0) {
+      // Serial.println("NEC");
+      irsend.sendNEC(0x1FE32CD);
+    }
+  }
+
+void setup() {
+#if defined(BLYNK_PRINT)
+  // Debug console
+  Serial.begin(115200);

```

Comment:
I've changed this from 9600 to 115200. Is that a problem at all?
I did this because all the other examples in this library use 115200.

Reply:
Not an issue, 115200 is fine.